### ISSUE TITLE
Export file to Nextcloud

### ISSUE THREAD

**Issue by Rindula** [ACTom/files_mindmap#107]
### Current Behavior
When exporting a Mindmap, the file will be downloaded via the Browser

### Wanted Behavior
Save the exported File in the Same Folder as the Mindmapfile in the Nextcloud drive. Naming could be the same as the Filename, but with the correct extension of course. So `example.km` exports to `example.md`.

**Comment by ACTom** [comment]
This needs to take into account issues such as file renaming, and the next step will be to add the function of saving as。

**Comment by larsik78** [comment]
It would be great, if there would be the opportunity for exporting also the notes/ comments  - not only the knots. :_-) I Used to write textes by mindmap software and the notes were a very important tool for that.

Thank you!

**Comment by CodeAlDente** [comment]
Yes an export of comments is highly needed. I just experienced that they are not included, even in the PDF file. So I had to put them as sub-knots to display them. A special comment section would be great or by using them as footnotes like Google does with comments in sheets.
